packages/create-turbo/src/transforms/update-commands-in-readme.ts

```diff
@@ -0,0 +1,50 @@
+import path from "node:path";
+import fs from "node:fs/promises";
+import { TransformError } from "./errors";
+import type { TransformInput, TransformResult} from "./types";
+
+const meta = {
+  name: "update-commands-in-readme",
+};
+
+export async function transform(args: TransformInput): TransformResult {
+  const { prompts, example } = args;
+
+  const selectedPackageManager = prompts.packageManager;
+  const readmeFilePath = path.join(prompts.root, "README.md");
+  try {
+    // Read the content of the file
+    let data = await fs.readFile(readmeFilePath, "utf8");
+
+    // replace package manager
+    const updatedReadmeData = replacePackageManager(selectedPackageManager, data);    
+
+    // Write the updated content back to the file
+    await fs.writeFile(readmeFilePath, updatedReadmeData, "utf8");
+
+  } catch (err) {
+    throw new TransformError("Unable to update README.md", {
+      transform: meta.name,
+      fatal: false,
+    });
+  }
+  return { result: "success", ...meta };
+}
+
+function replacePackageManager(packageManager, text) {
```

Comment:
Hi @tknickman 
I'm sorry for not getting back to you sooner. I tried writing a test file but could not test it on my local. I have made all the changes you asked for except this Test File. Can you please guide me on this?

Reply:
Hi @tknickman,
In case you're unavailable, I could use some assistance with the test files. Could you please let me know if there are any other test files currently in use within the project? I'm eager to ensure my contributions make a meaningful impact. Thanks in advance for your help!